README.md

```diff
@@ -861,6 +859,54 @@ vars:
 
 Changing `max_depth_dag` number to a higher one might prevent the package from running properly on BigQuery and Databricks/Spark.
 
+
+### Configuring exceptions to the rules
+
+While the rules defined in this package are considered best practices, we realize that there might be exceptions to those rules and people might want to exclude given results to get passing tests despite not following all the recommendations.
+
+An example would be excluding all models with names matching with `stg_..._unioned` from `fct_multiple_sources_joined` as we might want to union 2 different tables representing the same data in some of our staging models and we don't want the test to fail for those models.
+
+The package offers the ability to define a seed called `dbt_project_evaluator_exceptions.csv` to list those exceptions we don't want to be reported. This seed must contain the following columns:
+- `fct_name`: the name of the fact table for which we want to define exceptions
```

Comment:
Thanks Grace! The coverage ones can't have exceptions as they are reporting "aggregated" values rather than a list of exceptions, 1 by line.

To make it clearer without going into too many details, I'd be keen to mention that, as of today, it is not possible to exclude specific models for all the `coverage` tests, but there are variables available to configure those to the particular users' needs.

Does it sound good?